### PR TITLE
Feature/instructor add course validation yup

### PR DESCRIPTION
**Pull request by Raj-04**
## Description
Done some validations for the add-course form

Fixes # (issue)

## Loom Video

Please paste a link here to a quick loom video walking through what you did in your PR.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js [hunk 1]
```diff
@@ -15,12 +15,18 @@ const InstructorAddCourseFormSchema = yup.object().shape({
     .required('You must specify a minimum age'),
   max_age: yup
     .number()
-    .min(7, 'must be older than min')
+    .min(7, 'must be older than 7')
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js:18]: good, much more descriptive. Previously was unnecessarily ambiguous.

File: src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js [hunk 2]
```diff
@@ -42,7 +48,36 @@ const InstructorAddCourseFormSchema = yup.object().shape({
...
+    .typeError('Must be a valid zoom link')
     .required('Please enter a zoom link!'),
+  day: yup
+    .string()
+    .typeError('Must select a Day')
+    .oneOf(
+      [
+        'Monday',
+        'Tuesday',
+        'Wednasday',
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js:60]:
```suggestion
        'Wednesday',
```

Comment (jmerz826) [reply src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js:60]: Wednesday.. always the trickiest day of the week to spell

File: src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js [hunk 3]
```diff
@@ -42,7 +48,36 @@ const InstructorAddCourseFormSchema = yup.object().shape({
...
+  course_type: yup
+    .string()
+    .typeError('Must select a course')
+    .oneOf(
+      ['CoderYoga', 'CoderCamp', 'CoderSitters'],
+      'Please select a Course'
+    ),
+  course_name: yup
+    .string()
+    .min(5, 'Too Short! Has to be More than 5 words')
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js:77]:
```suggestion
    .min(5, 'Too Short! Has to be More than 5 characters')
```

File: src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js [hunk 4]
```diff
@@ -42,7 +48,36 @@ const InstructorAddCourseFormSchema = yup.object().shape({
...
+    .max(50, 'Too Long! Has to be less than 50 words')
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js:78]:
```suggestion
    .max(50, 'Too Long! Has to be less than 50 characters')
```

File: src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js [hunk 5]
```diff
@@ -42,7 +48,36 @@ const InstructorAddCourseFormSchema = yup.object().shape({
...
+        'Thursday',
+        'Friday',
+        'Saturday',
+        'Sunday',
+      ],
+      'Please select a day'
+    ),
+  course_type: yup
+    .string()
+    .typeError('Must select a course')
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js:70]:
```suggestion
    .typeError('Must select a program')
```

File: src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js [hunk 6]
```diff
@@ -42,7 +48,36 @@ const InstructorAddCourseFormSchema = yup.object().shape({
...
+    .oneOf(
+      ['CoderYoga', 'CoderCamp', 'CoderSitters'],
+      'Please select a Course'
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js:73]:
```suggestion
      'Please select a Program'
```

**Review by jmerz826 — CHANGES_REQUESTED** [review]
Overall nice job, Raj. Few naming convention inconsistencies and a small spelling error are the only outstanding issues I see.

File: src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js [hunk 7]
```diff
@@ -42,7 +48,36 @@ const InstructorAddCourseFormSchema = yup.object().shape({
...
+    ),
+  course_name: yup
+    .string()
+    .min(5, 'Too Short! Has to be More than 5 characters')
+    .max(50, 'Too Long! Has to be less than 50 characters')
+    .required('Course Name is Required')
+    .typeError('Course Name is required'),
 });
```
Comment (sagun1151) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js:81]: Great use of yup validation think it covers all aspects of add course form

**Review by sagun1151 — APPROVED** [review]

**Review by ElchinNasirov — APPROVED** [review]

**Review by jmerz826 — APPROVED** [review]

**Review by Adamh1990 — APPROVED** [review]

**Pull request merged**